src/main/java/org/jenkinsci/plugins/workflow/support/pickles/ExecutorPickle.java

```diff
@@ -53,6 +61,10 @@
 
     private final Queue.Task task;
 
+    private final boolean isEphemeral;
+
+    static long TIMEOUT_WAITING_FOR_EPHMERAL_NODE = TimeUnit.MINUTES.toMillis(5);
```

Comment:
Typo. Also probably this should use `SystemProperties` (2.4+, but you ought to be on 2.7.x anyway) to let it be configurable at runtime.